src/clients/java/java_bindings.zig

```diff
@@ -762,48 +772,48 @@ fn emit_u128_batch_accessors(
         });
-    } else {
-        // Set array:
-        try buffer.writer().print(
-            \\    /**
-            \\     * @param {[param_name]s} an array of 16 bytes representing the 128-bit value.
-            \\     * @throws IllegalArgumentException if {{@code {[param_name]s}}} is not 16 bytes long.
-            \\     * @throws IllegalStateException if not at a {{@link #isValidPosition valid position}}.
-            \\     * @throws IllegalStateException if a {{@link #isReadOnly() read-only}} batch.
-            \\
-        , .{
-            .param_name = to_case(field.name, .camel),
-        });
+    }
 
-        if (mapping.docs_link) |docs_link| {
-            try buffer.writer().print(
-                \\     * @see <a href="https://docs.tigerbeetle.com/{[docs_link]s}{[field_name]s}">{[field_name]s}</a>
-                \\     */
-                \\
-            , .{
-                .docs_link = docs_link,
-                .field_name = field.name,
-            });
-        } else {
-            try buffer.writer().print(
-                \\     */
-                \\
-            , .{});
-        }
+    // Set array:
+    try buffer.writer().print(
+        \\    /**
+        \\     * @param {[param_name]s} an array of 16 bytes representing the 128-bit value.
+        \\     * @throws IllegalArgumentException if {{@code {[param_name]s}}} is not 16 bytes long.
+        \\     * @throws IllegalStateException if not at a {{@link #isValidPosition valid position}}.
+        \\     * @throws IllegalStateException if a {{@link #isReadOnly() read-only}} batch.
+        \\
+    , .{
+        .param_name = to_case(field.name, .camel),
+    });
 
+    if (mapping.docs_link) |docs_link| {
         try buffer.writer().print(
-            \\    {[visibility]s}void set{[property]s}(final byte[] {[param_name]s}) {{
-            \\        putUInt128(at(Struct.{[property]s}), {[param_name]s});
-            \\    }}
-            \\
+            \\     * @see <a href="https://docs.tigerbeetle.com/{[docs_link]s}{[field_name]s}">{[field_name]s}</a>
+            \\     */
             \\
         , .{
-            .visibility = if (is_private or is_read_only) "" else "public ",
-            .property = to_case(field.name, .pascal),
-            .param_name = to_case(field.name, .camel),
+            .docs_link = docs_link,
+            .field_name = field.name,
         });
+    } else {
+        try buffer.writer().print(
+            \\     */
+            \\
+        , .{});
     }
 
+    try buffer.writer().print(
+        \\    {[visibility]s}void set{[property]s}(final byte[] {[param_name]s}) {{
+        \\        putUInt128(at(Struct.{[property]s}), {[param_name]s});
+        \\    }}
+        \\
+        \\
+    , .{
+        .visibility = if (is_private or is_read_only) "" else "public ",
+        .property = to_case(field.name, .pascal),
+        .param_name = to_case(field.name, .camel),
+    });
+
```

Comment:
This diff looks more complicated than it actually is due to indentation -- it just pulls the `set[property](byte[])` implementation out of the `else` branch.